src/cli/initConfig/config.js.template.hbs

```diff
@@ -177,8 +177,14 @@ module.exports = {
         /* if true detect dependencies that only exist before typescript-to-javascript compilation */
         // , tsPreCompilationDeps: false
 
-        /* if true leave symlinks untouched, otherwise use the realpath */
-        // , preserveSymlinks: false
+        /* if true detect dependencies that only exist before typescript-to-javascript compilation */
+        // , tsPreCompilationDeps: false
```

Comment:
tsPreCompilationDeps is duplicate now - doing away with the preserveSymlinks thing :stop_sign:

Reply:
fixed in 72da854